client/styles.css

```diff
@@ -31,6 +31,8 @@ label {
 	position: fixed;
 	top: 30px;
 	right: 30px;
+	overflow: scroll;
+	height: 500px
```

Comment:
Instead of `height: 500px;` (which might still overflow if the window is small), `bottom: 30px;` will make sure it's always the height of the window.